posthog/exceptions.py

```diff
@@ -33,6 +33,11 @@ def __init__(self, feature: Optional[str] = None) -> None:
         )
 
 
+class Conflict(APIException):
```

Comment:
Crazy that there is no built in for this in Django 😅

Reply:
May be because we're a major version behind 🙈